docs/installing.rst

```diff
@@ -5,42 +5,28 @@ Installation
 System Requirements
 *******************
 
-ThreatResponse now requires python >= 3.4.  It may still work(ish) with Python 2.7 but is not recommended due
-to security problems with cryptography libraries in Python 2.7.
+ThreatResponse requires python >= 3.4.
 
-While aws_ir is written purely in python, some of the libraries used require additional system packages.
-
-Fedora / RHEL Distributions
----------------------------
-
-* python-devel (Python 3.4+)
-* python-pip
-* libffi-devel
-* libssl-devel
-
-Debian Distributions
---------------------
-
-* python-dev (Python 3.4+)
-* python-pip
-* libffi-dev
-* libssl-dev
 
 Installing from PyPi
 ********************
 
 .. code-block:: bash
-
+   $ virtualenv env -p python3
```

Comment:
Should we be consistent and align this virtualenv instruction with the above?  `python3 -m virtualenv env`